### PR TITLE
SD-560 Migrate PDP

### PR DESCRIPTION
**Pull request by tomurb**

### CODE REVIEW

File: frontend/app/controllers/spree/products_controller.rb [hunk 1]
```diff
@@ -11,16 +13,16 @@ def index
       @searcher = build_searcher(params.merge(include_images: true))
       @products = @searcher.retrieve_products
       @products = @products.includes(:possible_promotions) if @products.respond_to?(:includes)
-      @taxonomies = load_taxonomies
```
Comment (damianlegawiec) [inline frontend/app/controllers/spree/products_controller.rb]: @tomurb we need to keep `load_taxonomies` usage here

File: frontend/app/controllers/spree/products_controller.rb [hunk 2]
```diff
@@ -11,16 +13,16 @@ def index
...
+      @taxonomies = Spree::Taxonomy.includes(root: :children)
+      @option_types = Spree::OptionType.includes(:option_values)
```
Comment (damianlegawiec) [inline frontend/app/controllers/spree/products_controller.rb]: @tomurb let's move this to `load_options` method

File: frontend/app/controllers/spree/products_controller.rb [hunk 3]
```diff
@@ -58,9 +77,5 @@ def redirect_if_legacy_path
         redirect_to url_for(params), status: :moved_permanently
       end
     end
-
-    def load_taxonomies
```
Comment (damianlegawiec) [inline frontend/app/controllers/spree/products_controller.rb]: @tomurb we need to keep this method in the same place

File: frontend/spec/features/product_page_spec.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
```
Comment (damianlegawiec) [inline frontend/spec/features/product_page_spec.rb]: @tomurb we don't need this spec, there's a lot of other specs in spree frontend gem for this action

File: frontend/app/controllers/spree/products_controller.rb (hunk 3 above)
Comment (tomurb) [reply frontend/app/controllers/spree/products_controller.rb]: it's above

**Pull request merged**